server/src/test/scala/cromwell/engine/EngineFunctionsSpec.scala

```diff
@@ -5,7 +5,7 @@ import org.scalatest.{FlatSpec, Matchers}
 class EngineFunctionsSpec extends FlatSpec with Matchers {
 // TODO WOM: move to wdl4s ?
 //  trait WdlStandardLibraryImpl extends  ReadLikeFunctions with WriteFunctions with PureStandardLibraryFunctionsLike {
-//    private def fail(name: String) = Failure(new NotImplementedError(s"$name() not implemented yet"))
+//    private def fail(name: String) = Failure(new UnsupportedOperationException(s"$name() not implemented yet"))
```

Comment:
I have to assume these tests are all removable since they've been sitting commented out for... years?

Reply:
Future PR? 😉